### DISCUSSION TITLE
使用qcefview调用网页地址，拖拽无效？

### DISCUSSION THREAD

**Discussion by junhunlrw** [CefView/QCefView#205] [Q&A]
用这个UI框架调用了一个网页，如果单纯在浏览器访问网页，里面的拖拽效果是可用的，但是放到这个框架引入之后，拖拽效果失效了，请问什么原因，是不是不兼容浏览器的drapstart、drapstop这些事件？

**Comment by tishion** [comment]
Drag&Drop是需要实现的，QCefView目前没有实现这个接口
